src/App.jsx

```diff
@@ -1,35 +1,107 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import React, { useState } from "react";
+import "./index.css";
+import QuoteText from "./components/QuoteText";
+import QuoteButton from "./components/QuoteButton";
+
+const quotes = [
```

Comment:
Just thought of one other thing - when we have a lot of JSON data like this, it's better to put it in it's own .json file and import it instead of put it in the main code's file assigned to a variable. Let me know if you need help figuring this out.